index.html

```diff
@@ -65,6 +65,124 @@ <h2 class="i-m">
             </li>
         </ul>
     </main>
+    <section class="works">
```

Comment:
- [ ] Please properly center the content in the work section to match the Figma. This will improve the look of your project & make it more attractive to employers. 😃 
![pages](https://github.com/AndresEspi/portafolio/assets/99656687/06434e7c-1860-4a67-b587-fbf95a88c4df)